### ISSUE TITLE
[Feature Request]: Auto completion of blade templates from Laravel 3rd party packages

### ISSUE THREAD

**Issue by abishekrsrikaanth** [laravel-idea/plugin#410]
### Feature Description

Can you add a feature to auto-complete blade templates defined on 3rd Party laravel packages? The default blade templates like

```
@if
@can
@unless
```

etc show on autocompletion, but blade templates defined on the packages don't show on auto-completion

**Comment by adelf** [comment]
Do you mean blade directives? Could you provide an example of it in the 3rd party packages?

**Comment by abishekrsrikaanth** [comment]
Correct. This is one of the packages that I use on every app I build.

Package: https://github.com/spatie/laravel-permission
Blade Docs: https://spatie.be/docs/laravel-permission/v5/basic-usage/blade-directives

Some sort of auto-discovery of all blade directives from all the packages will be very helpful.
